Header header

uint8 enabled_distortion
uint8 malicious_anchor

uint8 enabled_module
